packages/infer-schema/tests/integration/graphql/nodes.test.ts

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as neo4j from "neo4j-driver";
+import { toGraphQLTypeDefs } from "../../../src/index";
+import createDriver from "../neo4j";
+
+describe("GraphQL - Infer Schema nodes basic tests", () => {
+    const dbName = "inferToNeo4jGrahqlTypeDefsITDb";
+    let driver: neo4j.Driver;
+    const sessionFactory = (bm: string[]) => () =>
+        driver.session({ defaultAccessMode: neo4j.session.READ, bookmarks: bm, database: dbName });
+
+    beforeAll(async () => {
+        driver = await createDriver();
+        const cSession = driver.session({ defaultAccessMode: neo4j.session.WRITE });
+        try {
+            await cSession.writeTransaction((tx) => tx.run(`CREATE DATABASE ${dbName}`));
+        } catch (e) {
+            // ignore
+        }
```

Comment:
Are these tests going to work in Aura?

Reply:
Nope, they will not work in non-multidb env. I forgot to put that check in. Thanks!